keyboards/zinc/keymaps/default/keymap.c

```diff
@@ -0,0 +1,364 @@
+      _______, _______, _______, _______, _______, _______,                   _______, _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY \
+      ),
+
+  /* Adjust (Lower + Raise)
+   * ,-----------------------------------------.             ,-----------------------------------------.
+   * |      | Reset|RGBRST|Aud on|Audoff|      |             |      |Qwerty|Colemk|Dvorak|      | Ins  |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |RGB ON| HUE+ | SAT+ | VAL+ | Mac  |             | Win  |  -   |   =  |Print |ScLock|Pause |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      | MODE | HUE- | SAT- | VAL- |      |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      | EISU | EISU | EISU |             | KANA | KANA | Home |PageDn|PageUp| End  |
+   * `-----------------------------------------'             `-----------------------------------------'
+   */
+    [_ADJUST] =  LAYOUT( \
+      _______, RESET,   RGBRST,  AU_ON,   AU_OFF,  _______,                   _______, QWERTY,  COLEMAK, DVORAK,  _______, KC_INS, \
+      _______, RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, AG_NORM,                   AG_SWAP, KC_MINS, KC_EQL,  KC_PSCR, KC_SLCK, KC_PAUS,\
+      _______, RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, _______,                   _______, _______, _______, _______, _______, _______,\
+      _______, _______, _______, EISU,    EISU,    EISU,                      KANA,    KANA,    KC_HOME, KC_PGDN, KC_PGUP, KC_END\
+      )
+};
+
+
+
+#ifdef AUDIO_ENABLE
+
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+float tone_dvorak[][2]     = SONG(DVORAK_SOUND);
+float tone_colemak[][2]    = SONG(COLEMAK_SOUND);
+float tone_plover[][2]     = SONG(PLOVER_SOUND);
+float tone_plover_gb[][2]  = SONG(PLOVER_GOODBYE_SOUND);
+float music_scale[][2]     = SONG(MUSIC_SCALE_SOUND);
+#endif
+
+// define variables for reactive RGB
+bool TOG_STATUS = false;  
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+// Setting ADJUST layer RGB back to default
+void update_tri_layer_RGB(uint8_t layer1, uint8_t layer2, uint8_t layer3) {
+  if (IS_LAYER_ON(layer1) && IS_LAYER_ON(layer2)) {
+    #ifdef RGBLIGHT_ENABLE
+      rgblight_mode_noeeprom(RGB_current_config.mode);
+    #endif
+    layer_on(layer3);
+  } else {
+    layer_off(layer3);
+  }
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
+          PLAY_SONG(tone_qwerty);
+        #endif
+        persistent_default_layer_set(1UL<<_QWERTY);
```

Comment:
Could you replace this function?  

It's been depreciated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

```suggestion
        set_single_persistent_default_layer(_QWERTY);
```